### PR TITLE
Message Container ViewModel

### PR DESCRIPTION
**Pull request by BarAgam**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/teamagam/GimelGimel/pull/62" target="_blank">ReviewNinja

File: app/src/main/java/com/teamagam/gimelgimel/app/model/ViewsModels/messages/ContainerMessagesViewModel.java [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+            MessagesModel messagesModel) {
+        super(selectedMessageModel);
+        mMessagesReadStatusModel = messagesReadStatusModel;
+        mMessageModel = messagesModel;
+
+        enableObserverNotifyingOnModelChanges();
+    }
+
+    public boolean isMessageSelected() {
+        return mSelectedMessageModel.isAnySelected();
```
Comment (teamagam1) [inline app/src/main/java/com/teamagam/gimelgimel/app/model/ViewsModels/messages/ContainerMessagesViewModel.java:29]: isMessageSelected name is confusing. 
maybe isAnyMessageSelected

File: app/src/main/java/com/teamagam/gimelgimel/app/model/ViewsModels/messages/ContainerMessagesViewModel.java [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+    }
+
+    public int getUnreadMessageCount() {
```
Comment (teamagam1) [inline app/src/main/java/com/teamagam/gimelgimel/app/model/ViewsModels/messages/ContainerMessagesViewModel.java:32]: maybe also getTotalMessageCount?

Comment (BarAgam) [inline app/src/main/java/com/teamagam/gimelgimel/app/model/ViewsModels/messages/ContainerMessagesViewModel.java:32]: All in good time

File: app/src/main/java/com/teamagam/gimelgimel/app/model/ViewsModels/messages/ContainerMessagesViewModel.java (hunk 1 above)
Comment (BarAgam) [inline app/src/main/java/com/teamagam/gimelgimel/app/model/ViewsModels/messages/ContainerMessagesViewModel.java:29]: done

**Pull request merged**